data/scripting/infrastructure.lua

```diff
@@ -218,13 +245,19 @@ end
 -- RST
 -- .. function:: check_trees_rocks_poor_hamlet(player, sf, warehouse, waretable_rocks, waretable_trees)
```

Comment:
I think that this function would benefit from a hint in the doc why it has "poor hamlet" in its name